packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -175,6 +178,9 @@ export class FluidDataStoreRuntime extends EventEmitter implements IFluidDataSto
     private boundhandles: Set<IFluidHandle> | undefined;
     private _attachState: AttachState;
 
+    // NOTE: Search blob concept
```

Comment:
Same here - please add more on point comment describing higher-level concept (i.e. that it's not limited to search)